src/components/Player.vue

```diff
@@ -227,11 +227,7 @@
             backgroundImage: `url(${
               reminder.image && grimoire.isImageOptIn
                 ? reminder.image
-                : require(
-                    '../assets/icons/' +
-                      (reminder.imageAlt || reminder.role) +
-                      '.png',
-                  )
+                : `https://raw.githubusercontent.com/tomozbot/botc-icons/refs/heads/main/WEBP/${reminder.imageAlt || reminder.role}.webp`
```

Comment:
I really don't want to use Github as the image host, its not meant to be used as a CDN.  We could have the build step pull the images instead.